src/next-appdir/zz_internal/start.ts

```diff
@@ -14,8 +16,39 @@ export function startReactDsfr(params: {
     verbose?: boolean;
     /** Default: <a /> */
     Link?: (props: RegisteredLinkProps & { children: ReactNode }) => ReturnType<React.FC>;
+    /**
+     * When true, the nonce of the script tag will be checked, fetched from {@link DsfrHead} component and injected in react-dsfr scripts.
+     *
+     * @see https://developer.mozilla.org/fr/docs/Web/HTML/Global_attributes/nonce
+     */
+    checkNonce?: boolean;
```

Comment:
Would you consider performing a search and replace for doCheckNonce?
I don't intend to be overly nitpicky, but I aim to enforce a strict naming convention for boolean variables. The convention uses names that clearly indicate questions, such as isXxx, doesXxx, areXxx, etc.  

checkNonce could serve as the name of a function responsible for performing nonce verification.

There is an exception to this naming rule for component props. For instance, <Button disabled /> is more readable than <Button isDisabled />.